www/__tests__/footprintHelper.test.ts

```diff
@@ -9,8 +9,9 @@ import { getConfig } from '../js/config/dynamicConfig';
 import { mockBEMUserCache } from '../__mocks__/cordovaMocks';
 import { mockLogger } from '../__mocks__/globalMocks';
 import fakeLabels from '../__mocks__/fakeLabels.json';
+import fakeConfig from '../__mocks__/fakeConfig.json';
 
-mockBEMUserCache();
+mockBEMUserCache(fakeConfig);
```

Comment:
This is not your change, but I would like to understand this better. Why do we pass in the config instead of calling `putRWDocument`